### ISSUE TITLE
[Issue]: rockband controller output not working on wii

### ISSUE THREAD

**Issue by NikonUK** [sanjay900/guitar-configurator#16]
### Please describe your issue.

im not sure if its even supported but i have set the controller output type to be wii rockband guitar. the device does not show up on my nintendo wii. only doing this so i can replay through rockband on my original wii instead of emulators

### Version

yes

### What operating system are you using?

Other

### os-other

Wii (Homebrewed)

### What microcontroller are you using?

pi pico

### How is your controller programmed?

Wii

**Comment by sanjay900** [comment]
Are you using disk or a USB loader

**Comment by NikonUK** [comment]
I am using usbloadergx, doesn't even get detected with a homebrew input tester

**Comment by NikonUK** [comment]
I'll give it another go soon. Ill let you know next time as it might just of been User error but still unsure why it didn't work

**Comment by sanjay900** [comment]
> I am using usbloadergx, doesn't even get detected with a homebrew input tester

It won't do, rock band guitars aren't real Wii controllers

**Comment by NikonUK** [comment]
Unsure, I assume whichever one is default as I havnt changed it? I'll have to check. Do I need a custom IOS for the controller to work?

**Comment by NikonUK** [comment]
> > I am using usbloadergx, doesn't even get detected with a homebrew input tester
> 
> It won't do, rock band guitars aren't real Wii controllers
> 
> What IOS are you using?

The input tester detects Wii controllers, guitars, gamecube controllers and pro controllers, dk if it detects rb controllers or gh controllers but it just says guitar controllers

**Comment by sanjay900** [comment]
> > > I am using usbloadergx, doesn't even get detected with a homebrew input tester
> > 
> > 
> > It won't do, rock band guitars aren't real Wii controllers
> > What IOS are you using?
> 
> The input tester detects Wii controllers, guitars, gamecube controllers and pro controllers, dk if it detects rb controllers or gh controllers but it just says guitar controllers

Yeah, it will be talking about GH ones as those are actual Wii controllers.
